livy/src/main/java/org/apache/zeppelin/livy/LivySparkSQLInterpreter.java

```diff
@@ -80,7 +80,8 @@ public InterpreterResult interpret(String line, InterpreterContext interpreterCo
               line.replaceAll("\"", "\\\\\"")
                   .replaceAll("\\n", " ")
               + "\").show(" +
-              property.get("zeppelin.livy.spark.sql.maxResult") + ")",
+              property.get("zeppelin.livy.spark.sql.maxResult") + "," +
+              property.get("zeppelin.livy.spark.sql.truncateResult") + ")",
           interpreterContext, userSessionMap);
```

Comment:
I don't see whether `zeppelin.livy.spark.sql.maxResult` or any other property is validated in any place. What's special about `zeppelin.livy.spark.sql.truncateResult`? Moreover, in my opinion, if user doesn't read documentation, and sets something completely wrong - it is his own problem.